packer/bsd/netbsd_vanilla.pkr.hcl

```diff
@@ -0,0 +1,113 @@
+variable "bucket" { type = string }
+variable "gcp_project" { type = string }
+variable "image_date" { type = string }
+variable "image_name" { type = string }
+
+variable "prefix" {
+  type = string
+  default = ""
+}
+
+source "qemu" "qemu-gce-builder" {
+
+  boot_command            = [
+      "<enter><wait6s>",
+      "<enter><wait6s>",
+      "<enter><wait6s>",
+      "<down><enter><wait6s>",
```

Comment:
Might be worth adding a few comments explaining what all these waits are about...